### ISSUE TITLE
Bloquear voto da própria pessoa no link que enviou

### ISSUE THREAD

**Issue by gabrnunes** [gabrnunes/chuvis.co#8]

**Comment by rghedin** [comment]
Uma alternativa simples para “resolver” essa questão é ignorar. Hacker News e Reddit, salvo engano, computam automaticamente o voto de quem postou.

**Comment by gabrnunes** [comment]
Verdade, posso fazer um esquema pra pessoa votar automaticamente. Mas, dai, não faz sentido eu deixar a pessoa escolher se quer votar ou não @rghedin ?

**Comment by galenodemelo** [comment]
Acho que gera uma complexidade desnecessária. Do ponto de vista lógico, se a pessoa compartilhou a notícia é porque ela acha que merece ser destacada. Eu iria pela abordagem do @rghedin e computaria automaticamente.
